tensorflow/training/docker/2.0.2/py3/Dockerfile.gpu

```diff
@@ -0,0 +1,174 @@
+FROM nvidia/cuda:10.0-base-ubuntu18.04
+
+LABEL maintainer="Amazon AI"
+
+# prevent stopping by user interaction
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+ENV SAGEMAKER_TRAINING_MODULE sagemaker_tensorflow_container.training:main
+
+ENV PYTHONDONTWRITEBYTECODE=1 
+ENV PYTHONUNBUFFERED=1 
+ENV PYTHONIOENCODING=UTF-8 
+ENV LANG=C.UTF-8 
+ENV LC_ALL=C.UTF-8
+
+# Set environment variables for MKL
+# For more about MKL with TensorFlow see:
+# https://www.tensorflow.org/performance/performance_guide#tensorflow_with_intel%C2%AE_mkl_dnn
+ENV KMP_AFFINITY=granularity=fine,compact,1,0 
+ENV KMP_BLOCKTIME=1 
+ENV KMP_SETTINGS=0
+
+ARG PYTHON=python3
+ARG PYTHON_PIP=python3-pip
+ARG PIP=pip3
+
+ARG TF_URL=https://tensorflow-aws.s3-us-west-2.amazonaws.com/2.0.2/AmazonLinux/gpu/final/tensorflow_gpu-2.0.2-cp36-cp36m-manylinux2010_x86_64.whl
+
+RUN apt-get update && apt-get install -y --no-install-recommends --allow-unauthenticated \
+    python3-dev \
```

Comment:
Could you try to upgrade python version to 3.6.10? We need this version which is compatible with latest openssl 1.1.1g.

Please do the upgrade for both training and inference py3 images